qml/EventUtils.qml

```diff
@@ -78,6 +78,18 @@ QtObject{
             // E.g. "Daily; until 12/12/2014."
             str = i18n.tr("%1; until %2").arg(recurrence).arg(rule.limit.toLocaleString(Qt.locale(), dateFormat))
         }
+	
+	if (rule.interval !== undefined) {
```

Comment:
Don't use tabs, it breaks the indentation

Reply:
Oh, sry for the tabs! Normally my tabs are expanded to 4 spaces... I have to copy my .vimrc to that laptop...

Reply:
done